### DISCUSSION TITLE
Not in THOUSANDS OF HOURS HAS THIS HAPPENED AND IT'S THIS MODS FAULT (please read)

### DISCUSSION THREAD

**Discussion by Nerdgamerdave** [TEHE-Studios/ExpandedHelicopterEvents#364] [Q&A]
BRUH...

BEST moment in PZ I have had in literally MONTHS!  (so not a troll post :P).

I was in my base.  washing clothing, doing base stuff.  No fear of zombies anywhere.

Police heli flying.  He's blue.  No biggie.

CRASH!!

INTO.  MY.  HOUSE.

No weapons, barely wearing any armor of any kind. and the zombified chopper pilot, and his crew plop into the upper floor of my house and come down the stairs for me.

Chef's kiss.

Huck.  You the man.

-Foxy Roxy never panicked this bad in AGES.

![copterOnHouse](https://github.com/user-attachments/assets/d067eb3a-a01a-4093-aaf6-a03931d753d2)

**Comment by Chuckleberry-Finn** [comment]
<3
